### DISCUSSION TITLE
彩蛋卡池未开放

### DISCUSSION THREAD

**Discussion by azurstar** [Arondight/Adachi-BOT#511] [Q&A] [answered]
![截图](https://c2cpicdw.qpic.cn/offpic_new/1401068666//1401068666-1277266177-978EFB38BE751F864640DD0A6CFC5A01/0?term=255)

**Comment by Arondight** [comment, marked as answer]
忘了还有这么个卡池了，下午处理一下

**Reply by Arondight** [reply]
更新一下代码就好了
